### PR TITLE
Vet Clinic: Update Animals table with transaction

### PR DESCRIPTION
**Pull request by raihan2bd**
# Vet Clinic Database 🐻 
Hello

### Hi There 👋 In this PR;
- [ ] I added a `species` column inside the animals table ✅ 
01  species

- [ ] Then I `Insert` More data inside animals table ✅ 
02  Insert_more_data_to_the_animals_table

- [ ] Then Rollback species column with `unspecified` ✅ 
03  Species_rollback

- [ ] Then Inside a transaction I update animals table species column by name ✅ 
04  update_species_by_name

- [ ] Inside a transaction I deleted all records in the animals table, then roll back the transaction ✅ 
05 delete_animals_data_and_rollback

- [ ] Inside a transaction: ✔️ 
    - Deleted all animals born after Jan 1st, 2022.
    - Created a savepoint for the transaction.
    - Updated all animals' weight to be their weight multiplied by -1.
    - Rollbacked to the savepoint
    - Updated all animals' weights that were negative to be their weight multiplied by -1.
    - Committed transaction
06  update_animals_table_by_weight_and_birth_date

- [ ] Then added a couple of queries to select and count animals table data ✅ 
07  following_questions_part_1
08  following_questions_part_2

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -8,3 +8,6 @@ CREATE TABLE animals (
     neutered BOOLEAN,
     weight_kg Decimal
 );
+
+ALTER TABLE animals
+ADD species VARCHAR
```
Comment (VICTOR-CHUKWU) [inline schema.sql]: - [ ] nice work, kindly make this a Varchar with limit i.e `varchar(255)` as it is required in the project

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @raihan2bd  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (raihan2bd) [reply schema.sql]: @VICTOR-CHUKWU Thanks a lot for your valuable suggestion. 🙏  I will apply your suggestion before the next code review 👍

**Review by danmainah — APPROVED** [review]
Hi @raihan2bd , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

✔️ Gitflow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**